crates/swift-bridge-build/src/generate_core.rs

```diff
@@ -121,21 +127,27 @@ fn vec_of_primitive_headers(rust_ty: &str, c_ty: &str) -> String {
 
     // __private__OptionU8 ... etc
     let option_ty = format!("{}{}", "__private__Option", capatilized_first_letter);
+    let option_ty_tag = if option_ty == "__private__OptionF64" {
```

Comment:
We can delete these changes and change `OptionDouble` to `OptionF64` here https://github.com/chinedufn/swift-bridge/blob/20baa731361c9cb86c2b4a9f5c7d0a5f65f6ff45/crates/swift-bridge-build/src/generate_core.rs#L88

Reply:
Yeah, I've thought the same, just wasn't sure if there was a reason for them to not be equal. Anyway, will do this change and add the tests. Cheers

Reply:
Added tests [60af50d](https://github.com/chinedufn/swift-bridge/pull/200/commits/60af50d9ba3089ac8c63efd26b082a4c43dcd23f) and renamed `OptionDouble` to `OptionF64` [6dd8e42](https://github.com/chinedufn/swift-bridge/pull/200/commits/6dd8e420a38ea3edadc6f3018d188ad406adb440)